sqlalchemy_utils/view.py

```diff
@@ -124,7 +128,8 @@ def create_view(
     name,
     selectable,
     metadata,
-    cascade_on_drop=True
+    cascade_on_drop=True,
+    replace=False,
```

Comment:
Apologies, I just realized that the *replace* parameter needs to be documented in the function docstring. This will be picked up the next time the documentation is generated.

I think the parameter documentation can be added below `:param metadata:`. Perhaps something like:

```text
    :param replace:
        If ``True``, any view with the same *name* will be replaced.
        Note that materialized views cannot be replaced.
```

But please don't be limited by that suggestion!